### PR TITLE
 Translations Chinese

### PR DESCRIPTION
**Pull request by guyezi**
Translations Chinese

### CODE REVIEW

**Comment by aonez** [comment]
@guyezi thanks a lot! The files for 1.2.0 version are not yet final, that's why I'm not updating them on the repository yet.

I'll hold this PR until they're final and pushed in the repository.

File: Translations/zh-Hans.lproj/preferences.strings [hunk 1]
```diff
@@ -188,5 +203,23 @@
...
+"sjA-7B-5U6.title" = "运行最大压缩值:";
+
+/* Class = "NSTextFieldCell"; title = "Most operations use multiple threads. That means faster operations but also more power consumption and heat. "; ObjectID = "xyb-eH-aiy"; */
+"xyb-eH-aiy.title" = "大多数操作使用多线程.这意味着处理速度更快,但也会占用很多性能资源. ";
+
+/* Class = "NSButtonCell"; title = "Maximum simultaneous extractions:"; ObjectID = "yGo-Gm-0zq"; */
+"yGo-Gm-0zq.title" = "运行最大解压值:";
+
+/* Class = "NSTextFieldCell"; title = "Queing usually results in faster operations and a more responsive behaviour."; ObjectID = "zPq-ei-qUb"; */
+"zPq-ei-qUb.title" = "通常会导致更快的队列操作和更灵敏的行为.";
```
Comment (heshuimu) [inline Translations/zh-Hans.lproj/preferences.strings]: Suggestion:
“使用队列一般会使程序运行更快以及更灵敏。”

File: Translations/zh-Hans.lproj/preferences.strings [hunk 2]
```diff
@@ -188,5 +203,23 @@
...
 "sgV-AD-k4Q.title" = "启用外部卷访问权限";
+
+/* Class = "NSButtonCell"; title = "Maximum simultaneous compressions:"; ObjectID = "sjA-7B-5U6"; */
+"sjA-7B-5U6.title" = "运行最大压缩值:";
+
+/* Class = "NSTextFieldCell"; title = "Most operations use multiple threads. That means faster operations but also more power consumption and heat. "; ObjectID = "xyb-eH-aiy"; */
+"xyb-eH-aiy.title" = "大多数操作使用多线程.这意味着处理速度更快,但也会占用很多性能资源. ";
+
+/* Class = "NSButtonCell"; title = "Maximum simultaneous extractions:"; ObjectID = "yGo-Gm-0zq"; */
+"yGo-Gm-0zq.title" = "运行最大解压值:";
```
Comment (heshuimu) [inline Translations/zh-Hans.lproj/preferences.strings]: Suggestion:
“最多同时进行的解压缩个数：”

File: Translations/zh-Hans.lproj/preferences.strings [hunk 3]
```diff
@@ -188,5 +203,23 @@
...
+"otI-Pt-BTh.title" = "队列操作:";
+
 /* Class = "NSButtonCell"; title = "Enable external volumes access"; ObjectID = "sgV-AD-k4Q"; */
 "sgV-AD-k4Q.title" = "启用外部卷访问权限";
+
+/* Class = "NSButtonCell"; title = "Maximum simultaneous compressions:"; ObjectID = "sjA-7B-5U6"; */
+"sjA-7B-5U6.title" = "运行最大压缩值:";
+
+/* Class = "NSTextFieldCell"; title = "Most operations use multiple threads. That means faster operations but also more power consumption and heat. "; ObjectID = "xyb-eH-aiy"; */
+"xyb-eH-aiy.title" = "大多数操作使用多线程.这意味着处理速度更快,但也会占用很多性能资源. ";
```
Comment (heshuimu) [inline Translations/zh-Hans.lproj/preferences.strings]: Suggestion:
“大多数操作使用多个线程。这会使处理速度变得更快，但也会造成更多电力消耗和发热。”

File: Translations/zh-Hans.lproj/preferences.strings [hunk 4]
```diff
@@ -188,5 +203,23 @@
...
+"o8J-bR-bs6.title" = "运行最大运算值:";
+
+/* Class = "NSTextFieldCell"; title = "Queue operations:"; ObjectID = "otI-Pt-BTh"; */
+"otI-Pt-BTh.title" = "队列操作:";
+
 /* Class = "NSButtonCell"; title = "Enable external volumes access"; ObjectID = "sgV-AD-k4Q"; */
 "sgV-AD-k4Q.title" = "启用外部卷访问权限";
+
+/* Class = "NSButtonCell"; title = "Maximum simultaneous compressions:"; ObjectID = "sjA-7B-5U6"; */
+"sjA-7B-5U6.title" = "运行最大压缩值:";
```
Comment (heshuimu) [inline Translations/zh-Hans.lproj/preferences.strings]: Suggestion:
“最多同时进行的压缩个数：”

File: Translations/zh-Hans.lproj/preferences.strings [hunk 5]
```diff
@@ -188,5 +203,23 @@
 /* Class = "NSButtonCell"; title = "Use AES-256 when encrypting ZIP files (less compatible)"; ObjectID = "hp7-jO-HeV"; */
 "hp7-jO-HeV.title" = "加密 Zip 文件时使用 AES-256（兼容性较差）";
 
+/* Class = "NSButtonCell"; title = "Maximum simultaneous operations:"; ObjectID = "o8J-bR-bs6"; */
+"o8J-bR-bs6.title" = "运行最大运算值:";
```
Comment (heshuimu) [inline Translations/zh-Hans.lproj/preferences.strings]: Suggestion:
“最多同时进行的操作个数：”

File: Translations/zh-Hans.lproj/preferences.strings [hunk 6]
```diff
@@ -161,18 +161,33 @@
...
 "IHj-ac-103.title" = "启用完整访问权限（推荐）";
 
 /* Class = "NSTextFieldCell"; title = "Enable external volumes access so Keka can be used with your external disks, such as a USB or Thunderbolt drive:"; ObjectID = "JxT-Nq-8Z5"; */
 "JxT-Nq-8Z5.title" = "启用外部卷访问，以便 Keka 可以与外部磁盘一起使用，例如 USB 或 Thunderbolt：";
 
+/* Class = "NSTextFieldCell"; title = "Queue compression operations:"; ObjectID = "K5h-j5-bss"; */
+"K5h-j5-bss.title" = "队列压缩操作:";
+
+/* Class = "NSButtonCell"; title = "Inherit quarantine from downloaded files"; ObjectID = "KDR-ag-bll"; */
+"KDR-ag-bll.title" = "从下载的文件夹继承隔离";
```
Comment (heshuimu) [inline Translations/zh-Hans.lproj/preferences.strings]: Suggestion:
“从下载的文件继承隔离”

File: Translations/zh-Hans.lproj/preferences.strings [hunk 7]
```diff
@@ -161,18 +161,33 @@
 /* Class = "NSToolbarItem"; paletteLabel = "File Access"; ObjectID = "5eC-d6-04X"; */
 "5eC-d6-04X.paletteLabel" = "文件访问权限";
 
+/* Class = "NSTextFieldCell"; title = "Extractions use less power so even using multiple thread you can easily set this limit to two or three. Note that the target disk may suffer."; ObjectID = "DAr-Td-s5W"; */
+"DAr-Td-s5W.title" = "提取使用较少的功率,因此即使使用多线程,也可以轻松地将此限制设置为2或3.请注意,目标磁盘可能会受到影响.";
```
Comment (heshuimu) [inline Translations/zh-Hans.lproj/preferences.strings]: Suggestion:
“解压缩使用的电力较少，所以即使使用多线程，你也可以将此限制设置为2或3。请注意，目标磁盘可能会受影响。”

File: Translations/zh-Hans.lproj/preferences.strings [hunk 8]
```diff
@@ -161,18 +161,33 @@
...
+
 /* Class = "NSButtonCell"; title = "Enable home folder access"; ObjectID = "DL5-wY-cqc"; */
 "DL5-wY-cqc.title" = "启用主文件夹访问权限";
 
 /* Class = "NSTextFieldCell"; title = "Here you can choose the folders you want to give access to Keka. Note that Keka can't be used where it has no access. If needed, Keka will ask for specific folder access."; ObjectID = "GDf-ZS-dTs"; */
 "GDf-ZS-dTs.title" = "在这里，您可以选择要授予 Keka 访问权限的文件夹。请注意，Keka 无法在无法访问的地方使用。如果需要，Keka 将要求特定的文件夹访问权限。";
 
+/* Class = "NSTextFieldCell"; title = "It is recommended to only do one compression at a time if using multiple threads."; ObjectID = "Hkf-wB-gUj"; */
+"Hkf-wB-gUj.title" = "建议只使用一个线程一次进行一次压缩.";
```
Comment (heshuimu) [inline Translations/zh-Hans.lproj/preferences.strings]: Suggestion:
“如果使用多线程，建议不同时执行多个压缩操作。”

File: Translations/zh-Hans.lproj/preferences.strings [hunk 9]
```diff
@@ -161,18 +161,33 @@
...
+
 /* Class = "NSButtonCell"; title = "Enable full Mac access (Recommended)"; ObjectID = "IHj-ac-103"; */
 "IHj-ac-103.title" = "启用完整访问权限（推荐）";
 
 /* Class = "NSTextFieldCell"; title = "Enable external volumes access so Keka can be used with your external disks, such as a USB or Thunderbolt drive:"; ObjectID = "JxT-Nq-8Z5"; */
 "JxT-Nq-8Z5.title" = "启用外部卷访问，以便 Keka 可以与外部磁盘一起使用，例如 USB 或 Thunderbolt：";
 
+/* Class = "NSTextFieldCell"; title = "Queue compression operations:"; ObjectID = "K5h-j5-bss"; */
+"K5h-j5-bss.title" = "队列压缩操作:";
```
Comment (heshuimu) [inline Translations/zh-Hans.lproj/preferences.strings]: Please change punctuation ":" to "："

File: Translations/zh-Hans.lproj/preferences.strings [hunk 10]
```diff
@@ -161,18 +161,33 @@
...
+
+/* Class = "NSButtonCell"; title = "Inherit quarantine from downloaded files"; ObjectID = "KDR-ag-bll"; */
+"KDR-ag-bll.title" = "从下载的文件夹继承隔离";
+
+/* Class = "NSTextFieldCell"; title = "Limit number of threads per operation:"; ObjectID = "ND5-DN-C1H"; */
+"ND5-DN-C1H.title" = "限制每个操作线程数:";
```
Comment (heshuimu) [inline Translations/zh-Hans.lproj/preferences.strings]: Suggestion:
“限制每个操作使用的线程个数：”

File: Translations/zh-Hans.lproj/preferences.strings [hunk 11]
```diff
@@ -188,5 +203,23 @@
 /* Class = "NSButtonCell"; title = "Use AES-256 when encrypting ZIP files (less compatible)"; ObjectID = "hp7-jO-HeV"; */
 "hp7-jO-HeV.title" = "加密 Zip 文件时使用 AES-256（兼容性较差）";
 
+/* Class = "NSButtonCell"; title = "Maximum simultaneous operations:"; ObjectID = "o8J-bR-bs6"; */
+"o8J-bR-bs6.title" = "运行最大运算值:";
+
+/* Class = "NSTextFieldCell"; title = "Queue operations:"; ObjectID = "otI-Pt-BTh"; */
+"otI-Pt-BTh.title" = "队列操作:";
```
Comment (heshuimu) [inline Translations/zh-Hans.lproj/preferences.strings]: Please change punctuation ":" to "："

File: Translations/zh-Hans.lproj/Credits.html [hunk 12]
```diff
@@ -86,7 +86,7 @@
     
     
     <!-- START WARRANTY -->
-    <p class="warranty">THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.</p>
+    <p class="warranty">该软件由版权持有人和贡献者提供,如“是”和任何明示或暗示的保证,包括但不限于适销性和适合特定目的的默示保证被驳回.在任何情况下,版权持有人或贡献者均不对任何直接、间接、附带、特殊、惩戒性或后果性损害(包括但不限于替代货物或服务的采购;使用数据或利润的损失;或业务中断)承担责任,无论其原因如何,也不论其责任理论是否在合同中,因使用本软件而产生的严格责任或侵权行为(包括疏忽或其他),即使已告知此类损害的可能性.</p>
```
Comment (heshuimu) [inline Translations/zh-Hans.lproj/Credits.html]: I don't recommend translating the warranty notice unless this translation has been proof-read by a lawyer. The phrasing is not very natural, either.

**Comment by aonez** [comment]
@heshuimu note that this translations from an unfinished version that will change before the final release, so I'm holding it until then. Thanks por your tips!

File: Translations/zh-Hans.lproj/preferences.strings (hunk 1 above)
Comment (maxsky) [reply Translations/zh-Hans.lproj/preferences.strings]: Yes, your suggestion is right.

File: Translations/zh-Hans.lproj/preferences.strings (hunk 2 above)
Comment (maxsky) [reply Translations/zh-Hans.lproj/preferences.strings]: This is mine.
“最大同时解压数量：”

File: Translations/zh-Hans.lproj/preferences.strings (hunk 3 above)
Comment (maxsky) [reply Translations/zh-Hans.lproj/preferences.strings]: “大多数操作使用多线程。这使得处理速度在变快的同时也会增加功耗和发热。”

File: Translations/zh-Hans.lproj/preferences.strings (hunk 4 above)
Comment (maxsky) [reply Translations/zh-Hans.lproj/preferences.strings]: “最大同时压缩数量：”

File: Translations/zh-Hans.lproj/preferences.strings (hunk 5 above)
Comment (maxsky) [reply Translations/zh-Hans.lproj/preferences.strings]: “最大同时操作数量：”

File: Translations/zh-Hans.lproj/preferences.strings (hunk 6 above)
Comment (maxsky) [reply Translations/zh-Hans.lproj/preferences.strings]: agree ;)

File: Translations/zh-Hans.lproj/preferences.strings (hunk 7 above)
Comment (maxsky) [reply Translations/zh-Hans.lproj/preferences.strings]: “即使使用多线程，您也可以轻松地将此限制设置为 2 或 3 使得解压缩使用更少的功率。需注意的是目标磁盘可能会受到影响。”

File: Translations/zh-Hans.lproj/preferences.strings (hunk 8 above)
Comment (maxsky) [reply Translations/zh-Hans.lproj/preferences.strings]: “建议在使用多线程时依次压缩。”

File: Translations/zh-Hans.lproj/preferences.strings (hunk 10 above)
Comment (maxsky) [reply Translations/zh-Hans.lproj/preferences.strings]: Simpler might be better.
“限制每个操作的线程数：”

File: Translations/zh-Hans.lproj/Credits.html (hunk 12 above)
Comment (maxsky) [reply Translations/zh-Hans.lproj/Credits.html]: totally agree

**Pull request closed without merging**